bin/critterget.py

```diff
@@ -120,53 +149,72 @@ def apipost_with_response_code(uri, postdata=''):
         response = requests.post(url, headers=headers, data=pdata)
         if debug:
             print 'response is {}'.format(response.text)
-        return (response.status_code, response.json())
+        return response.status_code, response.json()
 
     except requests.exceptions.RequestException as e:
-        print u'{} MessageType="ApteligentError" Apteligent API retuned an error code: {} for the call to {}.  Maybe an ENTERPRISE feature?'.format(DATETIME_OF_RUN, e, url)
+        print u'{} MessageType="ApteligentError" Apteligent API retuned an ' \
+              u'error code: {} for the call to {}.  ' \
+              u'Maybe an ENTERPRISE feature?'.format(DATETIME_OF_RUN, e, url)
         return "ERROR"
 
+
 def apipost(uri, attribs=None):
     http_code, data = apipost_with_response_code(uri, attribs)
     return data
 
+
 def scopetime():
     # return an ISO8601 timestring based on NOW - Interval
     newtime=(datetime.datetime.utcnow() - datetime.timedelta(minutes=interval))
 
-    return (newtime.isoformat())
+    return newtime.isoformat()
 
 
 def getAppSummary():
-# read app summary information.  Print out in Splunk KV format.  Return a dict with appId and appName.
+    # read app summary information.
+    # Print out in Splunk KV format.
+    # Return a dict with appId and appName.
 
-    # list of the attributes we'll hand to Apteligent, also the list we will create as Splunk Keys
-    summattrs = "appName,appType,crashPercent,dau,latency,latestAppStoreReleaseDate,latestVersionString,linkToAppStore,iconURL,mau,rating,role,appVersions"
-    atstring = "attributes=%s"%summattrs
-    gdata = apicall("apps", atstring)
+    params = {'attributes': ','.join(SUMMARY_ATTRIBUTES)}
+    summary_data = apicall("apps", params)
 
     AppDict = {}
-    for appId in gdata.keys():
-        printstring = u'{} MessageType="AppSummary" appId={} '.format(DATETIME_OF_RUN, appId)
-        slist = summattrs.split(",")
-        for atname in slist:
-            printstring += u'{}="{}"'.format(atname,gdata[appId][atname])
-            if atname == "appName" : AppDict[appId]= {'name': gdata[appId][atname], 'versions': gdata[appId]['appVersions']}
+    for appId in summary_data[DATA].keys():
+        printstring = u'{} MessageType="AppSummary" appId={} '.format(
+            DATETIME_OF_RUN, appId
+        )
+
+        for attribute_name in SUMMARY_ATTRIBUTES:
+            printstring += u'{}="{}"'.format(
+                attribute_name,
+                summary_data[DATA][appId][attribute_name]
+            )
+            if attribute_name == "appName":
```

Comment:
I see lots of use cases for string literals here and elsewhere.  Please make an effort to clean them up.